src/types/JsonObject.ts

```diff
@@ -0,0 +1,5 @@
+import type { JsonObject } from 'swagger-ui-express';
+
+export const isJsonObject = (obj: unknown): obj is JsonObject => (
+  typeof obj === 'object'
```

Comment:
This is curious. Would we not want to conform to the actual `JsonObject` rather than our own definition of what it is? Or am I misunderstanding something here?

Reply:
(Ack sorry I thought you had approved)

This is basically the definition of JsonObject (insofar as an object in javsascript is defined as being something with `string` keys and `any` value)

Here's the definition of JsonObject:

```
export interface JsonObject {
    [key: string]: any;
}
```

Reply:
Oh there's a bug though (and I should write a unit test for it) -- `null` is the one case of something of type object that doesn't conform to the spec!  So this should actually be `typeof obj === 'object' && obj !== null`.

I'll open a fix PR to add that + some tests.

Reply:
(Sorry for the triple post)

Something I didn't explain here, but maybe part of the confusion: types / interfaces do not exist in runtime, so we can't use `JsonObject` directly.  The library does not provide type guards, unfortunately (they never do for some reason?? I don't understand why that isn't the norm.).  This means it is our responsibility to create the type guard that conforms with the interface.

This is a bit of a mind bend when moving from other typed languages where typing can often be enforced at runtime, and so a type and a type guard is the same thing.